pandas/tests/io/formats/test_format.py

```diff
@@ -3106,6 +3106,43 @@ def format_func(x):
         result = formatter.get_result()
         assert result == ["10:10", "12:12"]
 
+    def test_datetime64formatter_2d_array(self):
```

Comment:
do we need any tz-aware cases?

Reply:
Yes, good idea. It's likely to fail the same way so I'll add that too.

Reply:
done